tests/runner/src/test/resources/plugin-resources-resolver/mule-extension/META-INF/extension-manifest.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<extension-manifest>
```

Comment:
I would need to have an Extension, let me find out this in another PR so we don't hold these changes until this is done.